### PR TITLE
Fix cmakers

### PR DESCRIPTION
**Pull request by turtlesoupy**
Fixes CMake typos

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.0)
```
Comment (taylorgordon20) [inline CMakeLists.txt]: Did this work? Apparently he CUDA language thing was added in 3.8:

check_language(CUDA)
if(CMAKE_CUDA_COMPILER)

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,15 @@ build
 dist
 happy.egg-info
 demos/.ipynb_checkpoints
+
```
Comment (taylorgordon20) [inline .gitignore]: Hmm, I was hoping to ignore all of these files below at the directory level via the "build" ignore rule (i.e. they should be omitted by being under the build/ directory). Did CMake create a different directory for these files?

Comment (turtlesoupy) [reply .gitignore]: oh that might have been my doing

**Comment by turtlesoupy** [comment]
Can you take another look?

**Pull request merged**